keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/model/NamedImage.kt

```diff
@@ -12,28 +11,17 @@ data class NamedImage(
   val amis: Map<String, List<String>?>
 )
 
-class NamedImageComparator {
-
-  companion object : Comparator<NamedImage> {
-
-    override fun compare(a: NamedImage, b: NamedImage): Int =
-      (a.creationMs - b.creationMs).toInt()
-  }
```

Comment:
Just simplified some of the stuff in this class. Shouldn't make any difference to behavior